openmdao/components/bsplines_comp.py

```diff
@@ -110,6 +110,12 @@ def get_bspline_mtx(num_cp, num_pt, order=4, distribution='sine'):
 class BsplinesComp(ExplicitComponent):
     """
     Simple B-spline component for interpolation.
+
+    Attributes
+    ----------
+    cite : str
+        Listing of relevant citataions that should be referenced when publishing
```

Comment:
sp: citataions

Reply:
damn copy/paste of some ridiculous speller

Reply:
✅